Lib/test/test_uuid.py

```diff
@@ -640,8 +640,8 @@ def test_uuid5(self):
             equal(str(u), v)
 
     # TODO: RUSTPYTHON
-    @unittest.expectedFailureIfWindows("Fork is for Unix based systems only")
     @unittest.expectedFailure
+    @unittest.skipUnless(hasattr(os, 'fork'), "test needs os.fork()")
     @unittest.skipUnless(os.name == 'posix', 'requires Posix')
     def testIssue8621(self):
```

Comment:
The only decorator this has in CPython is `@support.requires_fork()`. I think this decorator should be sufficient.